base/Siege.php

```diff
@@ -91,12 +91,10 @@ protected function getArguments(): Vector<string> {
     if (!$this->options->noTimeLimit) {
       $arguments = Vector {
         // See Siege::getExecutablePath()  - these arguments get passed to
-        // 'timeout'
-        '-k', '5m',
         '5m',
         parent::getExecutablePath(),
       };
-    };
+    }
```

Comment:
It is GNU timeout, but it only supports -s. We could set it to just immediately send a sigkill rather than a sigterm then a sigkill with `--signal=9`